--- julia-0.6.0/base/filesystem.jl	2017-04-12 14:30:53.000000000 -0700
+++ patch-files/filesystem.jl	2017-04-14 15:33:23.000000000 -0700
@@ -52,7 +52,7 @@
 include("stat.jl")
 include("file.jl")
 include("poll.jl")
-include(string(length(Core.ARGS)>=2?Core.ARGS[2]:"","file_constants.jl"))  # include($BUILDROOT/base/file_constants.jl)
+include(string(length(Core.ARGS)>=2?Core.ARGS[2]:"","file_constants.jl"))  # include($JULIABUILDROOT/base/file_constants.jl)
 
 ## Operations with File (fd) objects ##
 
