--- julia-0.6.0/deps/llvm.mk	2017-04-12 14:30:53.000000000 -0700
+++ patch-files/llvm.mk	2017-04-13 16:15:57.000000000 -0700
@@ -528,6 +528,7 @@
 ifeq ($(LLVM_USE_CMAKE),1)
 
 $(LLVM_BUILDDIR_withtype)/build-configured: $(LLVM_SRC_DIR)/source-extracted | $(llvm_python_workaround) $(LIBCXX_DEPENDENCY)
+	patch ROLLPREFIX/SOURCEDIR/deps/srccache/llvm-3.9.1/lib/Transforms/Utils/LoopUtils.cpp ROLLPREFIX/patch-files/LoopUtils.cpp.patch
 	mkdir -p $(dir $@)
 	cd $(dir $@) && \
 		export PATH=$(llvm_python_workaround):$$PATH && \
