--- julia-0.6.0/deps/srccache/llvm-3.9.1.src/lib/Transforms/Utils/LoopUtils.cpp	2016-06-11 14:48:25.000000000 -0700
+++ patch-files/LoopUtils.cpp	2017-04-13 10:56:55.000000000 -0700
@@ -834,6 +834,12 @@
   return UsedOutside;
 }
 
+namespace llvm {
+  extern char &LoopSimplifyID;
+  extern char &LCSSAID;
+}
+
+
 void llvm::getLoopAnalysisUsage(AnalysisUsage &AU) {
   // By definition, all loop passes need the LoopInfo analysis and the
   // Dominator tree it depends on. Because they all participate in the loop
@@ -845,8 +851,6 @@
 
   // We must also preserve LoopSimplify and LCSSA. We locally access their IDs
   // here because users shouldn't directly get them from this header.
-  extern char &LoopSimplifyID;
-  extern char &LCSSAID;
   AU.addRequiredID(LoopSimplifyID);
   AU.addPreservedID(LoopSimplifyID);
   AU.addRequiredID(LCSSAID);
