--- julia-1.1.0/Make.inc	2019-01-12 16:25:11.000000000 -0800
+++ patch-files/Make.inc	2019-02-13 13:56:25.352966391 -0800
@@ -40,8 +40,8 @@
 USE_SYSTEM_DSFMT:=0
 USE_SYSTEM_BLAS:=0
 USE_SYSTEM_LAPACK:=0
-USE_SYSTEM_GMP:=0
-USE_SYSTEM_MPFR:=0
+USE_SYSTEM_GMP:=1
+USE_SYSTEM_MPFR:=1
 USE_SYSTEM_SUITESPARSE:=0
 USE_SYSTEM_LIBUV:=0
 USE_SYSTEM_UTF8PROC:=0
@@ -56,7 +56,7 @@
 
 ## Settings for various Intel tools
 # Set to 1 to use MKL
-USE_INTEL_MKL ?= 0
+USE_INTEL_MKL ?= 1
 # Set to 1 to use Intel LIBM
 USE_INTEL_LIBM ?= 0
 # Set to 1 to enable profiling with Intel VTune Amplifier
