--- julia-0.6.0/base/pcre.jl	2017-04-12 14:30:53.000000000 -0700
+++ patch-files/pcre.jl	2017-04-14 15:28:29.000000000 -0700
@@ -4,7 +4,7 @@
 
 module PCRE
 
-include(string(length(Core.ARGS)>=2?Core.ARGS[2]:"","pcre_h.jl"))  # include($BUILDROOT/base/pcre_h.jl)
+include(string(length(Core.ARGS)>=2?Core.ARGS[2]:"","pcre_h.jl"))  # include($JULIABUILDROOT/base/pcre_h.jl)
 
 const PCRE_LIB = "libpcre2-8"
 
