--- julia-0.6.0/base/sysimg.jl	2017-04-12 14:30:53.000000000 -0700
+++ patch-files/sysimg.jl	2017-04-14 15:30:32.000000000 -0700
@@ -154,8 +154,8 @@
 StridedVecOrMat{T} = Union{StridedVector{T}, StridedMatrix{T}}
 
 # For OS specific stuff
-include(string((length(Core.ARGS)>=2 ? Core.ARGS[2] : ""), "build_h.jl"))     # include($BUILDROOT/base/build_h.jl)
-include(string((length(Core.ARGS)>=2 ? Core.ARGS[2] : ""), "version_git.jl")) # include($BUILDROOT/base/version_git.jl)
+include(string((length(Core.ARGS)>=2 ? Core.ARGS[2] : ""), "build_h.jl"))     # include($JULIABUILDROOT/base/build_h.jl)
+include(string((length(Core.ARGS)>=2 ? Core.ARGS[2] : ""), "version_git.jl")) # include($JULIABUILDROOT/base/version_git.jl)
 
 include("osutils.jl")
 include("c.jl")
